From 55ab6f1389261edff5f4c942bc3b0d8e695856d7 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 30 Aug 2017 18:11:33 -0700
Subject: [PATCH 1/2] Add missing include path to -I options

Fixes errors like
| rwho.c:52:10: fatal error: 'protocols/rwhod.h' file not found
| #include <protocols/rwhod.h>
|          ^~~~~~~~~~~~~~~~~~~

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 rwho/Makefile  | 1 +
 rwhod/Makefile | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/rwho/Makefile b/rwho/Makefile
index 6f86388..67b28d4 100644
--- a/rwho/Makefile
+++ b/rwho/Makefile
@@ -6,6 +6,7 @@ include ../MRULES
 rwho: rwho.o
 	$(CC) $(LDFLAGS) $^ $(LIBS) -o $@
 
+CFLAGS += -I../include
 rwho.o: ../version.h
 
 install: rwho
diff --git a/rwhod/Makefile b/rwhod/Makefile
index 772b641..9034218 100644
--- a/rwhod/Makefile
+++ b/rwhod/Makefile
@@ -7,6 +7,8 @@ ifneq ($(USE_GLIBC),1)
 CFLAGS += -D_GNU_SOURCE
 endif
 
+CFLAGS += -I../include
+
 OBJS = rwhod.o
 
 rwhod: $(OBJS)
-- 
2.14.1

